package.json

```diff
@@ -21,11 +21,13 @@
         "typecheck": "tsc --noEmit",
         "format": "eslint --fix *.js *json 'src/**/*.+(ts|tsx|js|jsx|json|css)'",
         "linting": "eslint *.js *.json 'src/**/*.+(ts|tsx|js|jsx|json|css)'",
-        "validate": "npm run typecheck && npm run linting"
+        "validate": "npm run typecheck && npm run linting",
+        "setup_deckgl_types": "cd ./node_modules/@danmarshall/deckgl-typings/ && indefinitely-typed --folder deck.gl__aggregation-layers --folder deck.gl__core --folder deck.gl__extensions --folder deck.gl__geo-layers --folder deck.gl__google-maps --folder deck.gl__json --folder deck.gl__layers --folder deck.gl__mapbox --folder deck.gl__mesh-layers --folder deck.gl__react --folder deck.gl --folder luma.gl__constants --folder luma.gl__core --folder luma.gl__gltools --folder luma.gl__webgl-state-tracker --folder luma.gl__webgl --folder math.gl__core --folder math.gl"
     },
     "author": "R&T Equinor <akia@equinor.com>",
     "license": "LGPL",
     "dependencies": {
+        "@danmarshall/deckgl-typings": "^4.5.6",
```

Comment:
Is it necessary to have these as dependency or do they belong in dev-dependencies?

Reply:
I think the deckgl types need to be in dependencies, as they are used in parts exposed to the user, eg our ColormapLayerProps extend deckgl's BitmapLayerProps.

https://stackoverflow.com/questions/45176661/how-do-i-decide-whether-types-goes-into-dependencies-or-devdependencies